kvmagent/ansible/kvm.py

```
@@ -113,6 +113,111 @@ def update_libvirtd_config(host_post_info):
+
+    # Step 2: Check if the host already has a complete and valid cert set.
+    # Both server and client certs are required for TLS migration / V2V.
+    required_remote_files = [
+        "/etc/pki/CA/cacert.pem",
+        "/etc/pki/libvirt/servercert.pem",
+        "/etc/pki/libvirt/private/serverkey.pem",
+        "/etc/pki/libvirt/clientcert.pem",
+        "/etc/pki/libvirt/private/clientkey.pem",
+    ]
+    check_cmd = " && ".join(["test -f %s" % f for f in required_remote_files])
+    (status, _) = run_remote_command(check_cmd, host_post_info, return_status=True, return_output=True)
```

Ruff (0.15.5)

[error] `run_remote_command` may be undefined, or defined from star imports

(F405)